pdyrapi-kotlin-spring/build.gradle.kts

```diff
@@ -30,26 +30,36 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test")
 }
 
-openApiGenerate {
-    generatorName.set("kotlin-spring")
-    outputDir.set("$buildDir/generated")
-    ignoreFileOverride.set("$rootDir/.openapi-generator-ignore")
-    inputSpec.set("$rootDir/spec.yaml")
-    additionalProperties.set(mapOf("apiFirst" to "true"))
-    apiPackage.set("no.mattilsynet.api")
-    modelPackage.set("no.mattilsynet.model")
-    modelNameSuffix.set("Dto")
-    configOptions.set(mapOf(
-        "interfaceOnly" to "true",
-        "serializableModel" to "true",
-        "useBeanValidation" to "true",
-        "performBeanValuation" to "true",
-        "enumPropertyNaming" to "UPPERCASE"
-    ))
-    sourceSets.getByName(SourceSet.MAIN_SOURCE_SET_NAME).kotlin.srcDir("$buildDir/generated/src/main/kotlin")
-}
+val frontendsupportapi = "frontendsupportapi"
+val spec = "spec"
+
 
-tasks.register("openApiGenerate-task"){ dependsOn("openApiGenerate") }
+val openapiSpecs = mapOf(
+	spec to "spec.yaml",
+	frontendsupportapi to "reference/FrontendSupportAPI.yaml"
+)
+
+openapiSpecs.forEach {
+	tasks.create("openApiGenerate-${it.key}", org.openapitools.generator.gradle.plugin.tasks.GenerateTask::class) {
+		ignoreFileOverride.set("$rootDir/.openapi-generator-ignore")
+		generatorName.set("kotlin-spring")
+		additionalProperties.set(mapOf("apiFirst" to "true"))
+		inputSpec.set("$rootDir/${it.value}")
+		outputDir.set("$buildDir/generated")
+		apiPackage.set("no.mattilsynet.api")//${it.key}
```

Comment:
Fjerne utkommentert kode :)

Reply:
Du tenker på it.key?
Dette minnet meg på at jeg hadde kommentert ut hele skjemakomponent-blokka fra spec.yaml..